packages/graph/hash_graph/Makefile.toml

```
@@ -11,7 +11,7 @@ CARGO_MAKE_CARGO_PROFILE = "production"
 
 [tasks.test]
 run_task = [
-    { name = ["test-task", "yarn", "deployment-up", "recreate-db", "migrate-up", "test-integration", "test-rest-api", "deployment-down"], condition = { env_true = ["CARGO_MAKE_CI" ] } },
+    { name = ["test-task", "yarn", "deployment-up", "sleep-for-one-second", "migrate-up", "test-integration", "test-rest-api", "deployment-down"], condition = { env_false = ["CARGO_MAKE_CI" ] } },
```

This needs to be reverted
```suggestion
    { name = ["test-task", "yarn", "deployment-up", "sleep-for-one-second", "migrate-up", "test-integration", "test-rest-api", "deployment-down"], condition = { env_true = ["CARGO_MAKE_CI" ] } },
```

---

Good catch, thank you!